src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -1,5 +1,13 @@
 {% use "bootstrap_3_layout.html.twig" %}
 
+{% block form_label_class -%}
+col-sm-2
+{%- endblock form_label_class %}
+
+{% block form_group_class -%}
+col-sm-10
+{%- endblock form_group_class %}
```

Comment:
Same for this one. We try to not move snippets of code around as it makes merging branches more difficult.